samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -239,11 +239,19 @@ public async Task<string> AcquireExternalInformationAsync(SKContext context)
             var plan = Plan.FromJson(planJson, newPlanContext);
 
             // Invoke plan
-            SKContext planContext = await plan.InvokeAsync(plannerContext);
+            try
+            {
+                newPlanContext = await plan.InvokeAsync(plannerContext);
+            }
+            catch
```

Comment:
Please remove this try/catch. A try/catch with a 'throw' is the same as not having a try/catch at all.